logendpoint.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of the MAVLink Router project
+ *
+ * Copyright (C) 2017  Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <assert.h>
+
+#include "endpoint.h"
+#include "timeout.h"
+
+#define ALIVE_TIMEOUT 5
+#define TARGET_SYSTEM_ID 1
+#define SYSTEM_ID 2
```

Comment:
these defines shouldn't be in the header if you don't really need them here... move to .cpp.

If you need, you should either:
1) add appropriate "namespace", e.g. LOGENDPOINT_SYSTEM_ID
or
2) move it to be public `static const unsigned int` inside LogEndpoint